lstchain/io/tests/test_eventselection.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from lstchain.io import EventSelector
+
+def test_event_selection():
+   import pandas as pd
+   import numpy as np
+
+   evt_fil = EventSelector()
+
+   data = pd.DataFrame({"a": [1, 2, 3], "b": [2.2, 3.2, np.nan], "c": [1, 3, np.inf]})
```

Comment:
Does this test actually test that the cuts work? I just had a glimpse at the values and they seem all to be inside the ranges defined by the cuts, so this test is actually not testing if events are removed that are outside the cut intervals.

Also, you should probably also test with an astropy table as input, since you want to support both.

Reply:
Maybe define the data as an astropy table with units and then run the test once for the table and once for `table.to_pandas`.